common/schemas/keyman-touch-layout/README.md

```diff
@@ -84,6 +84,19 @@ optionally contain three additional properties: `font`, `fontsize`, and
 : A boolean value, if `true`, then the base layout's key cap will be displayed
   in the top left of the key cap. Defaults to `false`.
 
+* `displayHint`
+
+: A string value, one of `"none"`, `"dot"`, or `"text"`. Defaults to `"dot"`.
+  Controls the display of the hint on the key cap:
+
+  * `"none"`: show no hint on the key cap.
+  * `"dot"`: show a dot on top-right of key cap if a longpress menu
+    is available, default and only option for Keyman 15.0 and earlier.
+  * `"text"`: show the text `hint` property if present, often on
+    top-right of key cap.
```

Comment:
"Often on"?  What decides whether or not it is shown there?

Keeping in #4879 and #5079 both in mind, and the fact that we have traditionally displayed underlying key caps on the top-_left_ of a key, what's the plan for this behavior?  Will we drop the "top-left" part when using hints for underlying key caps, assuming the latter part of that clause?  Or will there be a way to select where it is displayed?

Reply:
Repositioning hints will be possible once we support theming (17.0 currently). It's all just CSS :grin: This text could just say 'on top-right' and leave the word 'often' out but figured it didn't hurt too much.

Reply:
And note that hints != underlying key. These are two separate strings that may *both* be displayed. The one thing that does go when we use hints is the "." marker indicating a longpress (see `displayHints` for how we control this).